### ISSUE TITLE
Would be REALLY GOOD to mention within SETUP Instructions.

### ISSUE THREAD

**Issue by obasekiosa** [koekeishiya/yabai#1913]
Would be really good to mention in the setup 

if you start yabai i.e. `yabai --start-service` and then disable accessibility it'll freeze your entire display/window.
Mouse movements still work, window selections still work but nothing else, not the dock, not selections, not the button clicks for anything outside window selection. absolutely nothing. The only way out is a system restart and hope (hopefully you didn't add it) `yabai --start-service` is not in any start-up script.

**Comment by anujc4** [comment]
This also happens everytime yabai is updated and started, because the newer version does not have the accessibility permissions.

**Comment by aragalie** [comment]
Yup, this has been also the case for the. Happens on EVERY update, to the point that I'm dreading doing 'brew upgrade`..... 🤦‍♂️

**Comment by koekeishiya** [comment]
This is a macOS permissions quirk; maybe just don't remove required permission from a piece of software *while it is still running*. Report a bug to Apple https://developer.apple.com/documentation/coregraphics/1454426-cgeventtapcreate?language=objc

Also the documentation literally says to stop yabai *before* upgrading and before uninstalling:
<img width="923" alt="Screenshot 2023-10-26 at 11 33 28" src="https://github.com/koekeishiya/yabai/assets/6175959/7fc541ea-adf7-4ccc-824a-9aee1f805988">

https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)#updating-to-the-latest-release

<img width="923" alt="Screenshot 2023-10-26 at 11 35 03" src="https://github.com/koekeishiya/yabai/assets/6175959/e3e38f50-315e-425c-83d1-03c14652c220">

https://github.com/koekeishiya/yabai/wiki/Uninstalling-yabai


> This also happens everytime yabai is updated and started, because the newer version does not have the accessibility permissions.

I don't have this issue on either on my machines (intel mbp2014 and newer m1 mbp) when using codesigned binaries.

**Comment by abdullah-kasim** [comment]
> Yup, this has been also the case for me. Happens on EVERY update, to the point that I'm dreading doing 'brew upgrade`..... 🤦‍♂️

@aragalie you could pin yabai using `brew pin koekeishiya/formulae/yabai` and use `brew list --pinned` to check if it's pinned.

**Comment by aragalie** [comment]
> > Yup, this has been also the case for me. Happens on EVERY update, to the point that I'm dreading doing 'brew upgrade`..... 🤦‍♂️
> 
> @aragalie you could pin yabai using `brew pin koekeishiya/formulae/yabai` and use `brew list --pinned` to check if it's pinned.
> 
> That way it won't upgrade automatically when you run `brew upgrade` so that you could do the yabai upgrade manually.

Thanks @abdullah-kasim I think this is actually the best solution to prevent this! `brew upgrade` automatically upgrading `yabai` without my control is definitely the culprit of this issue. 

With your pinning solution, when I would actually want to update `yabai`, I'll manually request brew update it, also making sure to first stop the service as @koekeishiya mentions above. 

@koekeishiya I think perhaps this advice 👆 could actually also be put in the setup instructions on the[Wiki](https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)#updating-to-the-latest-release)?
I would do a PR for it myself, but I'm not seeing a good way to contribute to the wiki...
